plasmapy/analysis/fit_functions.py

```diff
@@ -0,0 +1,1142 @@
+"""
+`FitFunction` classes designed to assist in curve fitting of swept Langmuir
+traces.
+"""
+__all__ = [
+    "AbstractFitFunction",
+    "Exponential",
+    "ExponentialPlusLinear",
+    "ExponentialPlusOffset",
+    "Linear",
+]
+
+import numpy as np
+
+from abc import ABC, abstractmethod
+from collections import namedtuple
+from scipy.optimize import curve_fit, fsolve
+from scipy.stats import linregress
+from typing import Tuple, Union
```

Comment:
Nicely alphabetized and sorted lists of imports always remind me of you!  :joy_cat: